### ISSUE TITLE
Edema localized around the clavicle

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10860]
**Preferred term label:**
Edema localized around the clavicle

**Synonyms**
Clavicular swelling, clavicular edema

**Definition (free text, please give PubMed ID)**
Swelling on the chest between the neck and the shoulder, above the collar bone

**Parent term (use hpo.jax.org/app)**
Related to https://hpo.jax.org/browse/term/HP:0006585 but more generalized as in diseases below.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital pseudoarthrosis of clavicle
Clavicle shaft fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001178
- label: Clavicular edema
- parents: [Edema (HP:0000969)](https://hpo.jax.org/app/browse/term/HP:0000969)
- pmid: 
- def: Swelling on the chest between the neck and the shoulder, above the clavicel (collar bone).
- comment: This condition may be observed with diseases including congenital pseudoarthrosis of clavicle and clavicle shaft fracture.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10860
- synonyms: Edema localized around the clavicle(NONE)|Clavicular swelling(NONE)
